From 86d4c2d229ec6f1a4b15b5393762775a565b96bf Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@gmail.com>
Date: Tue, 20 Nov 2018 15:05:17 +0800
Subject: [PATCH 07/25] drm: adi_axi_hdmi: fix compiling error due to the
 different base

drivers/gpu/drm/adi_axi_hdmi/axi_hdmi_drv.c:41:15: error: 'drm_fb_cma_create' undeclared here (not in a function); did you mean 'drm_gem_cma_create'
  .fb_create = drm_fb_cma_create,
               ^~~~~~~~~~~~~~~~~
drm_fb_cma_create has been removed in 4.18 due to unused, so replace it with
drm_gem_fb_create.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/adi_axi_hdmi/axi_hdmi_drv.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/adi_axi_hdmi/axi_hdmi_drv.c b/drivers/gpu/drm/adi_axi_hdmi/axi_hdmi_drv.c
index eec376aef158..1dc0ddcaac82 100644
--- a/drivers/gpu/drm/adi_axi_hdmi/axi_hdmi_drv.c
+++ b/drivers/gpu/drm/adi_axi_hdmi/axi_hdmi_drv.c
@@ -21,6 +21,7 @@
 #include <drm/drm.h>
 #include <drm/drm_crtc_helper.h>
 #include <drm/drm_gem_cma_helper.h>
+#include <drm/drm_gem_framebuffer_helper.h>
 #include <drm/drm_atomic_helper.h>
 
 #include "axi_hdmi_drv.h"
@@ -38,7 +39,7 @@ static void axi_hdmi_output_poll_changed(struct drm_device *dev)
 }
 
 static struct drm_mode_config_funcs axi_hdmi_mode_config_funcs = {
-	.fb_create = drm_fb_cma_create,
+	.fb_create = drm_gem_fb_create,
 	.output_poll_changed = axi_hdmi_output_poll_changed,
 	.atomic_check = drm_atomic_helper_check,
 	.atomic_commit = drm_atomic_helper_commit,
-- 
2.17.0

